Src/BootCamp.Chapter/Area51.cs

```diff
@@ -0,0 +1,204 @@
+                        Console.ReadKey();
+                        isKeepLoopingLogin = false;
+                    }
+                    else
+                    {
+                        isKeepLoopingLogin = AskTryAgain();
+                    }
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.Message);
+                    Console.ReadKey();
+                    isKeepLoopingLogin = AskTryAgain();
+                }
+                
+            }
+        }
+        private bool AskTryAgain()
+        {
+            bool keeploopingUntillYesOrNo = true;
+
+            while (keeploopingUntillYesOrNo)
+            {
+                Console.Clear();
+                Console.WriteLine("Wrong Login information!");
+                Console.WriteLine("Try Again? Y/N:");
+                string answer = Console.ReadLine().ToLower();
+
+                if (answer == "n")
+                {
+                    return false;
+                }
+                else if (answer == "y")
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        private void CreateAccount()
+        {
+            bool isInputGood = false;
+
+            while (!isInputGood)
+            {
+                try
+                {
+                    string name = AskForNameOrPasswordCreate("username");
+                    string password = AskForNameOrPasswordCreate("password");
+
+                    Credentials.TryParse(name + "," + password, out Credentials credentials);
+                    credentialsManager.Register(credentials);
+
+                    Console.WriteLine($"Account {name} created.");
+                    Console.ReadKey();
+
+                    isInputGood = true;
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.Message);
+                }
+            }
+        }
+
+        private string AskForNameOrPasswordCreate(string usernameOrPassword)
+        {
+            string input = string.Empty;
+            bool isInLoopForNameOrPassword = true;
+
+            while (isInLoopForNameOrPassword)
+            {
+                Console.Clear();
+                Console.WriteLine("Create Account.");
+                Console.Write($"Login {usernameOrPassword}:");
+                input = Console.ReadLine();
+
+                if (string.IsNullOrWhiteSpace(input))
+                {
+                    Console.WriteLine($"{usernameOrPassword} Cannot be empty.");
+                    Console.ReadKey();
+                }
+                else
+                {
+                    isInLoopForNameOrPassword = false;
+                }
+            }
+            return input;
+        }
+
+        private void See()
```

Comment:
See is a weird name, ambigious. What you do here is DisplayLoggedOnUsers

Reply:
totally right. i just thought i should name it the same as in the program but this is more readable.